src/identity-providers/add/DetailSettings.tsx

```diff
@@ -105,11 +108,15 @@ export const DetailSettings = () => {
   const form = useForm<IdentityProviderRepresentation>();
   const { handleSubmit, getValues, reset } = form;
   const [provider, setProvider] = useState<IdentityProviderRepresentation>();
+  const [selectedMapper, setSelectedMapper] =
+    useState<IdPWithMapperAttributes>();
 
   const adminClient = useAdminClient();
   const { addAlert, addError } = useAlerts();
   const history = useHistory();
   const { realm } = useRealm();
+  const [key, setKey] = useState(0);
+  const refresh = () => setKey(new Date().getTime());
```

Comment:
You can also increment this value by 1 to trigger the same behavior.
```suggestion
  const refresh = () => setKey(key + 1);
```